package.json

```diff
@@ -44,6 +44,7 @@
     "cookie-parser": "1.3.2",
     "copy-webpack-plugin": "3.0.1",
     "creditcards": "2.1.2",
+    "cross-env": "4.0.0-beta.0",
```

Comment:
If those dependencies are needed to make a `production` bundle, are they `devDependencies`? I always find this confusing on each different project :)